pkg/common/patterns.go

```diff
@@ -0,0 +1,56 @@
+package common
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+const EmailPattern = `\b([a-z0-9]{4,25}@[a-zA-Z0-9]{2,12}.[a-zA-Z0-9]{2,6})\b`
+const SubDomainPattern = `\b([0-9a-zA-Z]{2,40})\b`
+const UUIDPattern = `\b([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})\b`
+const UUIDPatternUpperCase = `\b([0-9A-Z]{8}-[0-9A-Z]{4}-[0-9A-Z]{4}-[0-9A-Z]{4}-[0-9A-Z]{12})\b`
+
+const RegexNumPattern = "0-9"
+const RegexPattern = "0-9a-z"
+const RegexPatternUpperCase = "0-9A-Z"
```

Comment:
These are not very descriptive and probably simple enough that they should be removed.